java/src/com/ibm/streamsx/topology/spl/SubmissionParameter.java

```diff
@@ -0,0 +1,26 @@
+package com.ibm.streamsx.topology.spl;
+
+import com.ibm.streamsx.topology.builder.BSubmissionParameter;
+
+public class SubmissionParameter extends BSubmissionParameter {
```

Comment:
I think this needs to be a class that contains a BSubmissionParameter rather than extending it.

Also need class level comments, at least indicating how this is used.